doc/api/cli.md

```diff
@@ -554,7 +554,7 @@ Enable the Permission Model for current process. When enabled, the
 following permissions are restricted:
 
 * File System - manageable through
-  [`--allow-fs-read`][], [`--allow-fs-write`][] flags
+  [`--allow-fs-read`][], [`--allow-fs-write`][] and [`--permission-fs-path-delimiter`][] flags
```

Comment:
I'm not sure it makes sense to add this here as this list is more about flags that restrict access and not general configuration.

Reply:
I was unsure too, I'll remove it